assets/javascripts/discourse/templates/components/admin-holidays-list.hbs

```diff
@@ -8,11 +8,11 @@
 
   <tbody>
     {{#each @holidays as |holiday|}}
-      {{admin-holidays-list-item
-        holiday=holiday
-        isHolidayDisabled=holiday.disabled
-        region_code=region_code
-      }}
+      <AdminHolidaysListItem
+        @holiday={{holiday}}
+        @isHolidayDisabled={{holiday.disabled}}
+        @region_code={{region_code}}
```

Comment:
huh, underscore case! 👀

Reply:
This is used in too many places to get cleaned up in this PR 😓, I will raise with the team though to have this cleaned up.